### PR TITLE
Authorization Tests with different gateway configurations

### PR DESCRIPTION
**Pull request by a-zuckut**

### CODE REVIEW

**Comment by a-zuckut** [comment]
@cmebarrow Could you please review these tests.

File: transport/wsn/src/test/scripts/org/kaazing/gateway/transport/wsn/auth/authorized.credentials.reponds.101.rpt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Normal gateway configuration with realm and authorization constraints
```
Comment (cmebarrow) [inline transport/wsn/src/test/scripts/org/kaazing/gateway/transport/wsn/auth/authorized.credentials.reponds.101.rpt]: Why are you adding any rpt files to gateway? Ideally all of the test scripts should be in the k3po specification project and the client side rpt's of the specification project should be used directly by the gateway ITs, i.e. I would expect to see only Java XxxIT classes in this pull request.

**Comment by a-zuckut** [comment]
@pkhanal Could you take a look at these gateway tests as well.

File: transport/http/src/test/webapp/public/resource [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+<head> 
+ <meta charset="utf-8">
+</head>
+
+<body>
+ <h1> THIS PAGE EXISTS!! </h1>
+</body>
+
+</html>
```
Comment (pkhanal) [inline transport/http/src/test/webapp/public/resource]: missing newline at the end of file

**Comment by cmebarrow** [comment]
@pkhanal @a-zuckut Did you talk to John about how we can avoid adding rpt files to gateway (see my comment above)? We're trying to get to the place where all our rpt scripts are in spec projects so I don't want us to do anything which moves us away from that.

**Comment by sbadugu** [comment]
@pkhanal @cmebarrow Can we take an action for this PR ?

**Comment by cmebarrow** [comment]
@sbadugu I have assigned it to me for now. More work will be needed to merge this, the rpt's should go into k3po/specification/ws and client side rpt's should be added. We'll need to create a story for this and schedule it into the next sprint if possible.

**Comment by sbadugu** [comment]
Thanks @cmebarrow . I see that you have already created a story for this. I have moved it to sprint backlog, but haven't assigned a milestone yet. Let's schedule it in the sprint planning meeting.

**Comment by sbadugu** [comment]
@AdrianCozma @NicoletaOita Can one of you please take a look at these tests. I want someone in QA to understand these tests and review it. Please comment on this PR after the review is done.

After it passes the review from QA, we can confirm from someone in HQ (CC @dpwspoon) and merge the PR.

**Comment by AdrianCozma** [comment]
@sbadugu Is this high priority? if not either me or @NicoletaOita will pick it up once we are done with tasks related to the EE 5.x vs 4.x certification

**Comment by sbadugu** [comment]
@AdrianCozma That's fine. We can come back to this later.

**Comment by sbadugu** [comment]
This is being covered by @danibusu in other PR.

**Pull request closed without merging**